From 6be20563c01c552262e4bb31c487315c6a678ba0 Mon Sep 17 00:00:00 2001
From: Ioana Radulescu <ruxandra.radulescu@nxp.com>
Date: Tue, 14 Jun 2016 17:23:13 +0300
Subject: [PATCH 263/388] staging: fsl-dpaa2/mac: Remove reference to MC
 firmware

Rephrase comment that was referring to MC as "firmware", as we
don't want to expose this view to the outside world.

Signed-off-by: Ioana Radulescu <ruxandra.radulescu@nxp.com>
[Original patch taken from SDK-V2.0-1703]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 drivers/staging/fsl-dpaa2/mac/mac.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/fsl-dpaa2/mac/mac.c b/drivers/staging/fsl-dpaa2/mac/mac.c
index e27a8d3..33cf888 100644
--- a/drivers/staging/fsl-dpaa2/mac/mac.c
+++ b/drivers/staging/fsl-dpaa2/mac/mac.c
@@ -120,7 +120,7 @@ static void dpaa2_mac_link_changed(struct net_device *netdev)
 		phy_print_status(phydev);
 	}
 
-	/* We must call into the MC firmware at all times, because we don't know
+	/* We must interrogate MC at all times, because we don't know
 	 * when and whether a potential DPNI may have read the link state.
 	 */
 	err = dpmac_set_link_state(priv->mc_dev->mc_io, 0,
-- 
2.9.3

